fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ProductAnalysisError.kt

```diff
@@ -49,50 +47,26 @@ fun ProductAnalysisError(
-                            tint = FirefoxTheme.colors.iconPrimary,
-                        )
-                    },
-                )
+            ProductReviewState.Error.GenericError -> ReviewQualityCheckInfoCard(
+                title = stringResource(id = R.string.review_quality_check_generic_error_title),
+                description = stringResource(id = R.string.review_quality_check_generic_error_body),
+                type = ReviewQualityCheckInfoType.Info,
+                modifier = Modifier.fillMaxWidth(),
+            )
 
-            ProductReviewState.Error.NetworkError ->
-                ReviewQualityCheckInfoCard(
-                    title = stringResource(id = R.string.review_quality_check_no_connection_title),
-                    description = stringResource(id = R.string.review_quality_check_no_connection_body),
-                    type = ReviewQualityCheckInfoType.Warning,
-                    modifier = Modifier.fillMaxWidth(),
-                    icon = {
-                        Icon(
-                            painter = painterResource(id = R.drawable.mozac_ic_warning_fill_24),
-                            contentDescription = null,
-                            tint = FirefoxTheme.colors.iconPrimary,
-                        )
-                    },
-                )
+            ProductReviewState.Error.NetworkError -> ReviewQualityCheckInfoCard(
+                title = stringResource(id = R.string.review_quality_check_no_connection_title),
+                description = stringResource(id = R.string.review_quality_check_no_connection_body),
+                type = ReviewQualityCheckInfoType.Warning,
+                modifier = Modifier.fillMaxWidth(),
+            )
 
-            ProductReviewState.Error.UnsupportedProductTypeError ->
-                ReviewQualityCheckInfoCard(
-                    title = stringResource(id = R.string.review_quality_check_not_analyzable_info_title),
-                    description = stringResource(id = R.string.review_quality_check_not_analyzable_info_body),
-                    type = ReviewQualityCheckInfoType.Info,
-                    modifier = Modifier.fillMaxWidth(),
-                    icon = {
-                        Icon(
-                            painter = painterResource(id = R.drawable.mozac_ic_information_fill_24),
-                            contentDescription = null,
-                            tint = FirefoxTheme.colors.iconPrimary,
-                        )
-                    },
-                )
+            ProductReviewState.Error.UnsupportedProductTypeError -> ReviewQualityCheckInfoCard(
+                title = stringResource(id = R.string.review_quality_check_not_analyzable_info_title),
+                description = stringResource(id = R.string.review_quality_check_not_analyzable_info_body),
+                type = ReviewQualityCheckInfoType.Info,
+                modifier = Modifier.fillMaxWidth(),
```

Comment:
optional: this could now instead be:

```
val title
val description
val type
when (error) { 
// set the above vals based on the error
}

ReviewQualityCheckInfoCard(
    // set the params
)
```

Reply:
nice! updated!